### PR TITLE
Liqun/update docs and plugins

### PR DESCRIPTION
**Pull request by liqul**
1. update various documents
2. config some plugins to disabled by default
3. add page browsing to web_search

### CODE REVIEW

File: project/plugins/sql_pull_data.py [hunk 1]
```diff
@@ -43,10 +45,11 @@ def __call__(self, query: str):
             SQL Query:"""
         prompt = ChatPromptTemplate.from_template(template)
 
-        db = SQLDatabase.from_uri(self.config.get("sqlite_db_path"))
+        if self.db is None:
```
Comment (ShilinHe) [inline project/plugins/sql_pull_data.py:48]: This is to avoid recreating a new SQLDB instance?

File: project/plugins/web_search.py [hunk 2]
```diff
@@ -9,6 +9,58 @@
...
+    top_k: int = 3,
+    chunk_size: int = 1000,
+    chunk_overlap: int = 250,
+) -> list[dict[str, Any]]:
+    try:
+        from langchain.text_splitter import RecursiveCharacterTextSplitter
+        from langchain_community.document_loaders import AsyncHtmlLoader
+        from langchain_community.document_transformers import Html2TextTransformer
+    except ImportError:
+        raise ImportError("Please install langchain-community first.")
```
Comment (ShilinHe) [inline project/plugins/web_search.py:24]: Also, langchain?

**Review by ShilinHe — APPROVED** [review]

File: project/plugins/sql_pull_data.py (hunk 1 above)
Comment (liqul) [reply project/plugins/sql_pull_data.py:48]: yes

File: project/plugins/web_search.py (hunk 2 above)
Comment (liqul) [reply project/plugins/web_search.py]: done

**Pull request merged**